traffic_ops/app/db/migrations/20200315000000_add_deliveryservice_service_category.sql

```diff
@@ -0,0 +1,25 @@
+/*
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+	    http://www.apache.org/licenses/LICENSE-2.0
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE deliveryservice ADD COLUMN service_category int REFERENCES service_category(id);
+CREATE TABLE IF NOT EXISTS service_category (
```

Comment:
doesnt the table need to be created prior to adding the reference in the deliveryservice table?